09-schema/03-rules.md

```diff
@@ -5,27 +5,64 @@ permalink: /docs/schema/rules
 
-When you query the knowledge graph for certain information, Grakn returns a complete set of answers. These answers contain explicit matches as well as implicit ones that have been inferred by the rules included in the schema.
+The rule-based reasoning allows automated capture and evolution of patterns within the knowledge graph. Graql reasoning is performed at query time and is guaranteed to be complete. Thanks to the reasoning facility, common patterns in the knowledge graph can be defined and associated with existing schema elements. The association happens by means of rules. This not only allows you to compress and simplify typical queries, but offers the ability to derive new non-trivial information by combining defined patterns.
+
+Once a given query is executed, Graql will not only query the knowledge graph for exact matches but will also inspect the defined rules to check whether additional information can be found (inferred) by combining the patterns defined in the rules. The completeness property of Graql reasoning guarantees that, for a given content of the knowledge graph and the defined rule set, the query result shall contain all possible answers derived by combining database lookups and rule applications.
 
 In this section, we learn more about how rules are constructed and how they are meant to be used.
 
-## Define a Rule
-Defining a Graql rule begins with a given label followed by `sub rule`, the `when` body as the condition, and the `then` body as the conclusion.
+## Defining Rules
+Graql rules assume the following general form:
+
+```
+when {rule-body}, then {rule-head};
+```
+
+People familiar with Prolog/Datalog, may recognise it as similar:
+
+```
+{rule-head} :- {rule-body}.
+```
+
+In Graql we refer to the body of the rule as the "when" part of the rule and the to head as the "then" part of the rule. Rules are integral members of the schema. As a result they require the define keyword for their creation. A rule is then defined by specifying its label followed by when and then blocks. Therefore, in Graql terms, we define rule objects in the following way:
 
 ```graql
-define
+define 
 
 rule-label sub rule,
-  when {
-    ## the condition
-  } then {
-    ## the conclusion
-  };
+when {
+    ##
+    ## conditions go here
+    ##
+},
+then {
+    ## concluded fact
+};
 ```
 
-Let's look an example.
+Each hashed line corresponds to a single Graql statement. In Graql, the "when" part of the rule is required to be a conjunctive pattern, whereas the "then" should be atomic - each rule can derive a single fact only. If your use case requires a rule with a disjunction in the "when" part, please notice that, when using the disjunctive normal form, it can be decomposed into series of conjunctive rules.
```

Comment:
let's stick to `we` and `our`. _If our use case requires ..._ as opposed to _If your use requires_. there are other places in this PR that `you` and `your` is used. please reflect this in change for those too